src/outputs.jl

```diff
@@ -28,6 +28,10 @@ function qqplot(output, results)
     for pvalue_col ∈ pvalue_cols
         pvalues = -log10.(filter(x -> !isnan(x), results[!, pvalue_col]))
         n = length(pvalues)
+        # If only 1 non-NA p-value, exit function
+        if n <= 1 
+            return
```

Comment:
Probably better to use "continue" instead of return here. This is because even  if one estimator has a sufficient set of values it will not be plotted in this implementation.

Reply:
Hi Olivier,

Thanks for looking at this! So no - basically a QQplot is unable to be generated with only a single p-value. Even if it is non-NA. I can update this to match the syntax you wrote here, as follows:

`n > 1 || continue`